framework/helpers/BaseHtml.php

```diff
@@ -735,8 +735,8 @@ public static function checkbox($name, $checked = false, $options = [])
      *
      * - groups: array, the attributes for the optgroup tags. The structure of this is similar to that of 'options',
      *   except that the array keys represent the optgroup labels specified in $items.
-     * - encodeSpaces: bool, whether to encode spaces in option prompt and option value with `&nbsp;` character.
```

Comment:
This option must stay to keep BC. Here is a possible logic work flow:

1. If `encodeSpaces` is set, ignore `encode`.
2. Otherwise check `encode`.